controllers/multiclusterobservability/observatorium.go

```diff
@@ -33,8 +34,48 @@ const (
 
 	readOnlyRoleName  = "read-only-metrics"
 	writeOnlyRoleName = "write-only-metrics"
+
+	ObservatoriumAPI     = "observatorium-api"
+	ThanosQuery          = "thanos-query"
+	ThanosQueryFrontend  = "thanos-query-frontend"
+	ThanosRule           = "thanos-rule"
+	ThanosReceive        = "thanos-receive-default"
+	ThanosStoreMemcached = "thanos-store-memcached"
+)
+
+var (
+	replicas1      int32 = 1
```

Comment:
seems we do not use this var.

Reply:
It is designed to be used by compact. Let me remove local variable in compact.